packages/api/src/StyleOptions.ts

```diff
@@ -943,6 +943,31 @@ type StyleOptions = {
    */
   feedbackActionsPlacement?: 'activity-actions' | 'activity-status';
 
+  fontColorPrimary?: string;
```

Comment:
Modify line 935 and say the feedback form only show if the `activity-actions`.

Also double check with designer.